play.lic

```diff
@@ -0,0 +1,43 @@
+custom_require.call %w[events]
+class Play
+  def initialize
+    @song ||= UserVars.song
+    check_flags
+    main_loop 
+  end
+
+  def clean?
+    return !Flags['clean']
+  end
+
+  def check_flags
+    Flags.add('play-clean', 'Your .* dirtiness may affect your performance') # Could check if the flag exists, but playing an instrument tells you if it needs cleaning
+    
+  end
+  
+  def play_song
+    fput("play #{@song} on zills")
+  end
```

Comment:
I noticed when using this previously that you can remove "on zills" without needing to say what instrument for those wearing cowbells or similar.

Reply:
Yea this is a super old script. I'll change that. I'll make a note in the comments too.